Build Python module for multiple Python installations. Patch is already submitted upstream and merged.
--- bpfcc.orig/src/python/CMakeLists.txt	2018-03-23 02:34:11.095202281 +0100
+++ bpfcc/src/python/CMakeLists.txt	2018-03-23 02:34:39.629462282 +0100
@@ -11,19 +11,24 @@
   set(PYTHON_CMD "python")
 endif()
 
-set(PIP_INSTALLABLE "${CMAKE_CURRENT_BINARY_DIR}/dist/bcc-${REVISION}.tar.gz")
 configure_file(setup.py.in ${CMAKE_CURRENT_BINARY_DIR}/setup.py @ONLY)
-# build the pip installable
-add_custom_command(OUTPUT ${PIP_INSTALLABLE}
-  COMMAND ${PYTHON_CMD} setup.py sdist
-  WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-  DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/bcc/__init__.py ${CMAKE_CURRENT_BINARY_DIR}/setup.py
-  )
-add_custom_target(bcc_py ALL DEPENDS ${PIP_INSTALLABLE})
-
 if(EXISTS "/etc/debian_version")
   set(PYTHON_FLAGS "${PYTHON_FLAGS} --install-layout deb")
 endif()
-install(CODE "execute_process(COMMAND ${PYTHON_CMD} setup.py install -f ${PYTHON_FLAGS}
-  --prefix=${CMAKE_INSTALL_PREFIX} WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})"
-  COMPONENT python)
+
+foreach(PY_CMD ${PYTHON_CMD})
+  string(REPLACE "/" "-" PY_CMD_ESCAPED ${PY_CMD})
+
+  set(PIP_INSTALLABLE "${CMAKE_CURRENT_BINARY_DIR}/dist-${PY_CMD_ESCAPED}/bcc-${REVISION}.tar.gz")
+  # build the pip installable
+  add_custom_command(OUTPUT ${PIP_INSTALLABLE}
+    COMMAND ${PY_CMD} setup.py sdist --dist-dir dist-${PY_CMD_ESCAPED}
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+    DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/bcc/__init__.py ${CMAKE_CURRENT_BINARY_DIR}/setup.py
+    )
+  add_custom_target(bcc_py_${PY_CMD_ESCAPED} ALL DEPENDS ${PIP_INSTALLABLE})
+
+  install(CODE "execute_process(COMMAND ${PY_CMD} setup.py install -f ${PYTHON_FLAGS}
+    --prefix=${CMAKE_INSTALL_PREFIX} WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})"
+    COMPONENT python)
+endforeach()
